external/ag-website-shared/src/components/link-icon/LinkIcon.tsx

```diff
@@ -1,12 +1,58 @@
 import { Icon } from '@ag-website-shared/components/icon/Icon';
 import classnames from 'classnames';
-import type { AllHTMLAttributes } from 'react';
+import { type AllHTMLAttributes, useEffect, useRef, useState } from 'react';
 
 import styles from './LinkIcon.module.scss';
 
 export function LinkIcon({ className, ...props }: AllHTMLAttributes<HTMLAnchorElement> & { children?: never }) {
+    const [linkCopied, setLinkCopied] = useState(false);
+    const [linkActive, setlinkActive] = useState(false);
+    const copiedTimeoutRef = useRef(null);
+    const activeTimeoutRef = useRef(null);
+
+    const onclick = (event) => {
+        event.preventDefault();
+
+        const href = event.target.href;
+
+        navigator.clipboard.writeText(href);
+
+        setLinkCopied(true);
+        setlinkActive(true);
+
+        if (copiedTimeoutRef.current) clearTimeout(copiedTimeoutRef.current);
+        if (activeTimeoutRef.current) clearTimeout(activeTimeoutRef.current);
+
+        copiedTimeoutRef.current = setTimeout(() => {
+            setLinkCopied(false);
+        }, 2250);
+
+        activeTimeoutRef.current = setTimeout(() => {
+            setlinkActive(false);
+        }, 2000);
+    };
+
+    useEffect(() => {
+        return () => {
+            if (copiedTimeoutRef.current) {
+                clearTimeout(copiedTimeoutRef.current);
```

Comment:
Shouldn't you clear the timeout for `activeTimeoutRef` too?

```suggestion
                clearTimeout(copiedTimeoutRef.current);
                clearTimeout(activeTimeoutRef.current);
```